### DISCUSSION TITLE
Fixing right stick

### DISCUSSION THREAD

**Discussion by qxb3** [AntiMicroX/antimicrox#1006] [Q&A]
Hi are there other methods of fixing my un-calibrated controller? the calibration is just not working for me.
![image](https://github.com/AntiMicroX/antimicrox/assets/59109346/80b8e1c5-8b4e-49ba-9676-273c4f2a038a)

**Comment by Phantoma1** [comment]
The right stick I have mouse down, mouse up, mouse left, mouse right.

**Reply by qxb3** [reply]
i meant that the right stick is not centered properly. see where the dot is

**Reply by Phantoma1** [reply]
Oh, then you need to recalibrate  the controller.
Try this, https://www.howtogeek.com/241421/how-to-calibrate-your-gaming-controller-in-windows-10/#why-would-i-need-to-do-this

So have you tried calibrating the controller and it hasn't worked?

**Reply by qxb3** [reply]
ye, i already looked at the windows built-in calibration tool and does not need to be calibrated since its already ok, the problem really is on the antimicrox itself, also tried the calibration there but its not centering.
